protobuf/flow/executiondata/executiondata.proto

```diff
@@ -25,4 +34,34 @@ message GetExecutionDataByBlockIDRequest {
 
 message GetExecutionDataByBlockIDResponse {
   entities.BlockExecutionData block_execution_data = 1;
+}
+
+message SubscribeExecutionDataRequest {
+  bytes start_block_id = 1;
+  uint64 start_block_height = 2;
```

Comment:
You may want to describe precedence of arguments and the behaviour when both of them are zero